src/Disciplina/DB.hs

```diff
@@ -0,0 +1 @@
+{-# OPTIONS_GHC -F -pgmF autoexporter #-}
```

Comment:
I'm against things like that, becase I think, reexporting module will define intentions more clear.

Reply:
Point taken, though we extensively used that in Cardano. I personally don't think that this is bad in cases where module subtree is simple and there's no need to not import some submodules in parent module - it saves time and simplifies adding new submodules. We should discuss it with the rest of the team and come up with some kind of best practice, written down somewhere.